requirements/requirements.txt

```diff
@@ -3,4 +3,4 @@ click-default-group~=1.2
 mycroft-messagebus-client~=0.10
```

Comment:
We need some method to create a client without loading the full config stack first. I had refactored to use ovos-bus-client but had to revert changes because it would take several seconds to instantiate a client object